docs/_theme/djangodocs/static/djangodocs.css

```diff
@@ -43,12 +43,13 @@ div.nav { margin: 0; font-size: 11px; text-align: right; color: #487858;}
 
 /*** basic styles ***/
 dd { margin-left:15px; }
-h1,h2,h3,h4,h5 { margin-top:1em; font-family:"Trebuchet MS",sans-serif; font-weight:normal; }
+h1,h2,h3,h4,h5,h6,h7,h8,h9,h10,h11,h12 { margin-top:1em; font-family:"Trebuchet MS",sans-serif; font-weight:normal; }
```

Comment:
Is there a reason to have a different font for the headings (this doesn't happen with the docs on the main docs page).

Reply:
TBH :shrug: Styles for docs.djangoproject.com are defined in the https://github.com/django/djangoproject.com/tree/master/djangoproject/scss. The use of `singlehtml` is so niche ...

Reply:
I actually just went digging for how the docs on the main page are built/ styled. I think there may be more value in having the docs here look more like they do on the main docs page 🤷.